calico-cloud/get-started/connect/usage-metrics.mdx

```diff
@@ -11,4 +11,11 @@ In Manager UI:
 1. From the cluster drop-down menu, select your cluster.
 1. Click the Profile icon, and select **Usage Metrics**.
 
-![usage-metrics](/img/calico-cloud/usage-metrics.png)
\ No newline at end of file
+![usage-metrics](/img/calico-cloud/usage-metrics.png)
+
+## How Node Hours are calculated
+
+`Node Hours` are calculated for a single node by dividing the vCPU of each node by four, rounding up, and then *adding them together*. Because the calculation is on a per-node basis, the results are not the same as adding up all the vCPU and dividing by four. `Node Hours` are updated every hour, and change every month because of the different number of days in a month.
+
+For example, if you install {{prodname}} on a single machine with 32 physical cores (64 cores are presented to the kernel), and if each node only goes up to 4 vCPU, the result is 64 vcpu / 4 = 16 billable nodes.
```

Comment:
This example is confusing as it starts out describing a single machine (node?), but then says "each node".

I think you meant "each node hour"?

But even that doesn't make sense because you say "if each node [hour] only goes up to 4vCPU" but there is no case in which it wouldn't, as that is the constant defined formula

```suggestion
For example, if you install {{prodname}} on a single machine with 32 physical cores (64 cores are presented to the kernel), and if each node hour only goes up to 4 vCPU, the result is 64 vcpu / 4 = 16 billable nodes.
```

Reply:
It sounds like this is not ready to document. Closing until PM approves.